src/features/game/events/landExpansion/collectCompost.test.ts

```diff
@@ -82,16 +82,16 @@ describe("collectComposterProduce", () => {
     ).toThrow("Compost is not ready");
   });
 
-  it("removes the Compost from the building", () => {
-    const basicComposter: PlacedItem = {
+  it("removes the Produce from the building", () => {
```

Comment:
Romy and I had this discussion a few times.

"Produce" means "crops brought to the grocery store".  The composters _consume_ that rather than create it.

The previous naming was better.